src/models/stores/nav-tabs.ts

```diff
@@ -10,7 +10,7 @@ export const NavTabsConfigModel = types
     showNavPanel: false
   })
   .views(self => ({
-    getNavTabSpec(tabId: ENavTab) {
+    getNavTabSpec(tabId: ENavTab | string) {
```

Comment:
I had to modify this so that "sort-work" could be accepted.

Reply:
Rather than doing this, you should update ENavTab to include "sort-work".

Reply:
Turns out `ENavTab` is an exported `enum` so I used `ENavTab.kSortWork`